src/Parser/Monk/Windwalker/Modules/Spells/FistsofFury.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+
+import Module from 'Parser/Core/Module';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+// Inspired by the penance bolt counter module from Discipline Priest
+
+const FISTS_OF_FURY_MINIMUM_TICK_TIME = 100; // This is to check that additional ticks aren't just hitting secondary targets
+
+class FistsofFury extends Module {
+    previousTickTimestamp = null;
+    fistsTickNumber = 0;
+    fistsCastNumber = 0;
+    averageTicks = 0;
+
+    isNewFistsTick(timestamp) {
+        return !this.previousTickTimestamp || (timestamp - this.previousTickTimestamp) > FISTS_OF_FURY_MINIMUM_TICK_TIME;
+    }
+
+    on_byPlayer_cast(event) {
+        const spellId = event.ability.guid;
+        if (spellId === SPELLS.FISTS_OF_FURY_CAST.id) {
```

Comment:
More of a code style,it would potentially easier to read if you moved the logic out of the `if` statement and wrote the statement like `if(spellId !== SPELLS.FIST_OF_FURY_CAST.id) { return; }` This just cleans up the code below if additional logic is added.